mods/ra2/rules/soviet-naval.yaml

```diff
@@ -64,6 +64,7 @@ sub:
 	Mobile:
 		TurnSpeed: 4
 		Speed: 71
+	-LeavesTrails:
```

Comment:
Should this be conditional based on whether or not the sub is submerged?

Reply:
In RA2, leaving trails had nothing to do with cloaking, but to Underwater=yes, which disabled trails altogether if present (along with some other effects like sub targeting for NavalTargeting modes).